docs/feature_oled_driver.md

```diff
@@ -0,0 +1,245 @@
+# OLED Driver
+
+## OLED Supported Hardware
+
+128x32 OLED modules using SSD1306 driver IC over I2C. Supported on AVR based keyboards. Possible but untested hardware includes ARM based keyboards and other sized OLED modules using SSD1306 over I2C, such as 128x64. 
+
+**WARNING:** This OLED Driver currently uses the new i2c_master driver from split common code. If your split keyboard uses i2c to communication between sides this driver will most likely cause a conflict (serial is fine). Please contact your keyboard vendor and ask them to migrate to the latest split common code to fix this. Message @mechmerlin#2999 in the qmk discord for more details. (ps: he likes rick-rolls)
```

Comment:
In theory, as long as the i2c address doesn't overlap.... it shouldn't be an issue.  But I suspect that you'd have to add some handling for this.  

Also, there was this: https://github.com/mtei/qmk_firmware/issues/7

Reply:
Ya, it was more a warning that I haven't tested it and many keyboards are using the old i2c source as well. That and a slight nudge to get more keyboards moved to split common. Interesting link, saved it to read later.

Reply:
I think a less dire warning would be better.

Also, `!>` will create a warning note. https://docs.qmk.fm/#/documentation_best_practices?id=important

Reply:
```suggestion
!> Warning: This OLED Driver currently uses the new i2c_master driver from split common code. If your split keyboard uses i2c to communication between sides, this driver will most likely cause an address conflict (serial is fine). Please contact your keyboard vendor and ask them to migrate to the latest split common code to fix this. Message @mechmerlin#2999 in the qmk discord for more details. (ps: he likes rick-rolls)
```

Reply:
Fixed